services/website/website.tester.js

```diff
@@ -4,14 +4,17 @@ const t = (module.exports = require('../tester').createServiceTester())
 
 t.create('status of http://shields.io')
   .get('/website.json?url=http://shields.io')
+  .timeout(7500)
   .expectBadge({ label: 'website', message: 'up', color: 'brightgreen' })
 
 t.create('status of https://shields.io')
   .get('/website.json?url=https://shields.io')
+  .timeout(7500)
   .expectBadge({ label: 'website', message: 'up', color: 'brightgreen' })
 
 t.create('status of nonexistent domain')
   .get('/website.json?url=http://shields.io.io')
+  .timeout(15000)
```

Comment:
Could we put the timeout on the service code instead? This does seem like a quite a bit bit of time.

Reply:
I'm not opposed to changing the behavior in the case of slow response times, whether that manifests as a new badge message or we assume the site is down after X seconds. IMO though that warrants some broader discussion and thought, and at the moment I'm just trying to get our service test runs back to a passing state.

In my local env the tests were usually taking between 6 and 10 seconds, but I figured it wouldn't hurt to play the timeout conservatively.

Reply:
Fair enough, will approve them.